### ISSUE TITLE
Manual accessibility testing and fixes

### ISSUE THREAD

**Issue by nguyen-tina** [cagov/unemployment#319]
- [x] Color contrast and font size (John)
- [X] Radio buttons (Tony)
- [X] headers usage "Just checked this item and it is now has the correct heading order.  Before the H2 was identified as ‘Week 1’ instead of ‘Important Next Steps’."
- [x] Form labels (Sharon) - only for the "Find Me", should be resolved by adding aria-label.
- [x] timing - "Reads through like the end of a sentence,  ‘Find me button’.  Adding an aria label to this button will help…more descriptive (missing form label)."

Additional Findings (sounds optional?):

- [ ] Making the timing clock adjustable is recommended. Currently after inactivity you will need to start over for security purposes.   https://www.w3.org/TR/2016/NOTE-UNDERSTANDING-WCAG20-20161007/time-limits-required-behaviors.html
- [ ] Date of birth fields allow for entering any characters. Example is provided in the header but that may not be sufficient. Recommended to add the format into the fields and to make number fields if that is the criteria to follow.
- [ ] Social Security number reads bullet for all numbers then option to ‘Show SSN’ is read.  Recommendation to have this information before user enters information.

**Comment by nguyen-tina** [comment]
I also wanted to share [this research](https://github.com/uswds/uswds/issues/1004) that the best case scenario would be to only label _optional_ fields, rather than required fields (since we have fewer optional fields than required). I recognize that this may be a constraint, due to matching styling and consistency with the current UIO process.

**Comment by idealisms** [comment]
Updated the original issue text to include updates from Shavana (accessibility tester).

**Comment by sharonwarner** [comment]
Shavana confirmed the headers, color contrast, form labels/errors, and radio buttons are all fixed. Only remaining issues will require coordination with ISO/EDD/PAB. Moving this back to backlog.
